projects/python-client/src/datapane/client/api/user.py

```diff
@@ -113,6 +116,76 @@ def _run_script(script: str):
     runpy.run_path(str(script_path), run_name="__datapane__")
 
 
+def _run_template(template_path: str):
+    """Run the template script"""
+    # Store the cwd before we change it
+    old_cwd = os.getcwd()
+    # Change cwd to support relative paths in template script
+    os.chdir(template_path)
```

Comment:
in `datapane.common.utils` there is the `pushd` contextmanager you can use to temporarily change dir, i.e.

```python
with pushd(template_path):
   ...
```

Reply:
Excellent - updated to use `pushd`.